src/Symfony/Component/Form/Extension/Core/Type/CheckboxType.php

```diff
@@ -33,6 +34,16 @@ public function buildForm(FormBuilderInterface $builder, array $options)
         // doing so also calls setDataLocked(true).
         $builder->setData(isset($options['data']) ? $options['data'] : false);
         $builder->addViewTransformer(new BooleanToStringTransformer($options['value'], $options['false_values']));
+
+        if (!method_exists($builder, 'setIsEmptyCallback')) {
+            @trigger_error(sprintf('Not implementing the setIsEmptyCallback() method in %s which implements %s is deprecated since Symfony 4.4.', \get_class($builder), FormConfigBuilderInterface::class), E_USER_DEPRECATED);
```

Comment:
same here

Reply:
sprintf(`'Not implementing the "%s::setIsEmptyCallback()" method in "%s" is deprecated since Symfony 4.4.', FormConfigBuilderInterface::class, \get_class($builder))`